src/wazuh_db/wdb_global.c

```diff
@@ -874,12 +871,15 @@ cJSON* wdb_global_select_groups(wdb_t *wdb) {
         return NULL;
     }
 
-    stmt = wdb->stmt[WDB_STMT_GLOBAL_SELECT_GROUPS];
+    sqlite3_stmt *stmt = wdb->stmt[WDB_STMT_GLOBAL_SELECT_GROUPS];
 
-    result = wdb_exec_stmt(stmt);
+    int sql_status = SQLITE_ERROR;
+    cJSON* result = wdb_exec_stmt_sized(stmt, WDB_MAX_RESPONSE_SIZE, &sql_status, STMT_MULTI_COLUMN);
```

Comment:
Wouldn't be better to use STMT_SINGLE_COLUMN here?

Reply:
As we discussed with the team, it'd be more efficient but it won't implemented here.